index.html

```diff
@@ -5,10 +5,183 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/4.0.0/normalize.css">
+  <link rel="stylesheet" 
+  href="https://cdnjs.cloudflare.com/ajax/libs/normalize/4.0.0/normalize.css">
   <link rel="stylesheet" href="dist/style.css">
 </head>
 <body>
+  <div class="page">
 
+    <header class="header">
+      <div class="header__logo logo">
+        <img class="logo__img" src="img/logo.png" alt="logotype">
+      </div>
+      <nav class="nav header__nav">
+        <a class="nav__link" href="#">LOG IN</a>
+        <a class="nav__link" href="#">SIGN UP</a>
+        <a class="nav__link" href="#">
+          <img src="img/setting.png" alt="setting">
+        </a>
+      </nav>
+      <div class="main-head">
+        <p class="header__p">Find your Favorite</p>
+        <h1 class="header__h1">
+          RECIPES
+        </h1>
+        <button class="main-head__button">GET STARTED</button>
+        <div class="header__txt">
+          OR SEND US YOUR OWN RECIPES AND GET REWARDED!
+        </div>
+      </div>
+    </header>
+
+    <section class="features">
+      <h2 class="h2_visible">Features</h2>
+      <article class="features__item">
+        <img class="features__img" src="img/inspired.png" alt="inspired">
+        <h3 class="features__header">GET INSPIRED</h3>
+        <hr class="features__line">
+        <p class="features__body">
+          Lorem ipsum dolor sit amet, consectetur adipisc 
+          Pellentesque vel enim a elit viverra elementuma.
+          Aliquam erat volutpat. 
+        </p>
+      </article>
+      <article class="features__item">
+        <img class="features__img" src="img/rewarded.png" alt="rewarded">
+        <h3 class="features__header">GET REWARDED</h3>
+        <hr class="features__line">
+        <p class="features__body">
+          Lorem ipsum dolor sit amet, consectetur adipisc 
+          Pellentesque vel enim a elit viverra elementuma.
+          Aliquam erat volutpat.  
+        </p>
+      </article>
+      <article class="features__item">
+        <img class="features__img" src="img/social.png" alt="get social">
+        <h3 class="features__header">GET SOCIAL</h3>
+        <hr class="features__line">
+        <p class="features__body">
+          Lorem ipsum dolor sit amet, consectetur adipisc 
+          Pellentesque vel enim a elit viverra elementuma.
+          Aliquam erat volutpat.            
+        </p>
+      </article>
+    </section>
+
+    <section class="app">
+      <h2 class="h2_visible">INTRODUCING THE FUDI APP</h2>
```

Comment:
you can use a modifier only along with element or block like this "h2 h2_visible"